op-chain-ops/genesis/db_migration.go

```diff
@@ -67,6 +67,15 @@ func MigrateDB(ldb ethdb.Database, config *DeployConfig, l1Block *types.Block, m
 		)
 	}
 
+	if uint64(config.L2OutputOracleStartingTimestamp) > l1Block.Time() {
```

Comment:
Is the L2 Output Oracle timestamp the time of the transition blocK? If so this check should be done the other way (transition block timestamp must be >= l1Block timestamp)

Reply:
I think this is underspecified - I'm not 100% sure what makes the most sense here. I thought that the deploy param `startingL2Timestamp` for the `L2OutputOracle` should be the same value as the timestamp in the genesis block

Reply:
This leads me to believe we shouldn't even have `L2OutputOracleStartingTimestamp` as part of the config

Reply:
We could check how it is used in the contract & see if we can relax the constraints there & pull from the L1 block time the L2 genesis time